docs/data-sources/ip_ranges.md

```diff
@@ -34,3 +30,14 @@ resource "aws_security_group" "from_fastly" {
 * `ipv6_cidr_blocks` - The lexically ordered list of ipv6 CIDR blocks.
 
 [1]: https://docs.fastly.com/guides/securing-communications/accessing-fastlys-ip-ranges
+<!-- schema generated by tfplugindocs -->
```

Comment:
There is currently some overlap between our own curated "Attributes Reference" and the "Schema" section generated dynamically by the new `tfplugindocs` tool, but until the issue (hashicorp/terraform-plugin-docs#28) is resolved we might want to hold off making any further content changes.

Reply:
After discussing with @phamann I've decided I'm going to review each individual file and manually decide whether the curated content should stay or go. This is because some of the generated documentation files have dynamically generated Schema data that is very good and ends up making our own "Attributes Reference" section completely redundant (i.e. lots of duplication).